.projenrc.ts

```diff
@@ -5,12 +5,12 @@ const project = new CdklabsConstructLibrary({
   projenrcTs: true,
   author: 'AWS',
   authorAddress: 'aws-cdk-dev@amazon.com',
-  cdkVersion: '2.45.0',
+  cdkVersion: '2.77.0',
```

Comment:
the bucket deployment process complained about the `addSource` call when building the bucket deployment object. upgrading the CDK version fixed that - but i'm not sure if this is truly required. but didn't think it would cause other harm.